src/pages/AzurePreview.js

```diff
@@ -225,6 +249,47 @@ const AzurePreviewUserForm = ({ value: formValue, onChange, onSubmit }) => {
         ),
       ]),
 
+      fieldset({ style: { padding: 0, border: 'none', margin: '1rem 0', width: '100%' } }, [
+        h(FormLabel, { htmlFor: 'azure-preview-interest-org-description', required: true }, [
+          'Which of the following best describes your organization?',
+        ]),
+        h(Select, {
+          id: 'azure-preview-interest-org-description',
+          getOptionLabel: ({ value }) => value,
+          value: formValue.organizationDescription,
+          onChange: (opt) => onChange({ ...formValue, organizationDescription: opt.value }),
+          options: organizationDescriptions,
+        }),
+
+        formValue.organizationDescription.includes('Other') &&
+          h(Fragment, [
+            h(FormLabel, { htmlFor: 'azure-preview-interest-org-description-other', required: true }, ['Please specify']),
+            h(ValidatedInput, {
+              inputProps: {
+                id: 'azure-preview-interest-org-description-other',
+                placeholder: 'Description',
+                value: formValue.organizationDescriptionOther,
+                onChange: (value) => {
+                  setFieldsTouched((v) => ({ ...v, organizationDescriptionOther: true }));
+                  onChange({ ...formValue, organizationDescriptionOther: value });
+                },
+              },
+              error: fieldsTouched.organizationDescriptionOther && !formValue.organizationDescriptionOther ? 'A description is required' : undefined,
+            }),
+          ]),
+      ]),
+
+      div({ style: { width: '100%' } }, [
+        h(FormLabel, { htmlFor: 'azure-preview-interest-org-region', required: true }, ['In which region is your organization located?']),
+        h(Select, {
```

Comment:
We should have these selects render above the Dropdown (or set to auto), as of right now it extends the screen significantly.

I'll see if I can find the snippet that can handle this...

![Screenshot 2023-06-27 at 1 16 14 PM](https://github.com/DataBiosphere/terra-ui/assets/11773357/471c56fe-3f9b-4c38-be46-82e11e1e08cd)

Reply:
Nice, didn't know about that option. Done in https://github.com/DataBiosphere/terra-ui/pull/3985/commits/e7bf064179d45ae637a2a765a622fc062876918f.